package.json

```diff
@@ -16,6 +16,8 @@
     "morgan": "^1.10.0"
   },
   "devDependencies": {
+    "axios": "^1.0.0",
```

Comment:
This should be dependencies, not `devDependencies`

Reply:
I fixed it